apps/webpack.config.js

```diff
@@ -12,6 +12,8 @@ const UnminifiedWebpackPlugin = require('unminified-webpack-plugin');
 const {WebpackManifestPlugin} = require('webpack-manifest-plugin');
 const WebpackNotifierPlugin = require('webpack-notifier');
 const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+const CircularDependencyPlugin = require('circular-dependency-plugin');
+const circles = require('./circular_dependencies.json');
```

Comment:
nit; I like that the file got renamed from `circles.json`, but I'd also like to see the variable name get updated to match, both here and below. A little bit of additional verbosity feels appropriate

```suggestion
const circlularDependencies = require('./circular_dependencies.json');
```